api/src/wrangler.rs

```diff
@@ -1,56 +1,92 @@
 use std::cell::RefCell;
 use std::collections::HashMap;
+use std::error::Error;
 
 use anyhow::anyhow;
+use bimap::BiBTreeMap;
+use cid::multihash::{Code, MultihashDigest};
+use cid::Cid;
+use fvm_shared::crypto::signature::{
+    Signature, SECP_PUB_LEN, SECP_SIG_LEN, SECP_SIG_MESSAGE_HASH_SIZE,
+};
+use fvm_shared::error::ExitCode;
+use fvm_shared::piece::PieceInfo;
+use fvm_shared::sector::RegisteredSealProof;
+use std::fmt;
+
+use fil_actors_runtime::runtime::builtins::Type;
+use fil_actors_runtime::runtime::{Policy, Primitives};
+use fil_actors_runtime::test_utils::{make_piece_cid, recover_secp_public_key};
```

Comment:
This code can just be copied here, I think. Eventually removed when we only have one version of the tests.